### DISCUSSION TITLE
Complete build options for Pthread API [8618]

### DISCUSSION THREAD

**Discussion by elfring** [eProsima/Fast-DDS#3459] [Ideas]
Would you like to add the configuration script “[AX_PTHREAD](https://www.gnu.org/software/autoconf-archive/ax_pthread.html "How to build C programs using POSIX threads")” to [your build specification](https://github.com/eProsima/Fast-RTPS/blob/c4abe694842acba10abbddbb8c63d179c2716344/configure.ac#L40 "Update candidate")?

**Comment by JLBuenoLopez** [comment]
Marking this as an enhancement that could be considered to be included in the future. Moving to the Ideas discussion forum as per [Fast DDS Contributing guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md#issues-and-support)
